sql/update/populate_customer_in_projects.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/*
+ * Copyright (C) 2016 Igalia, S.L. <info@igalia.com>
+ *
+ * This file is part of PhpReport.
+ *
+ * PhpReport is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PhpReport is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PhpReport.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * This script selects the customer id related to each project and populate the new customerid
+ * column in project table.
+ */
+
+define('DB_NAME', 'phpreport');
+define('DB_USER', 'phpreport');
+define('DB_PASSWORD', 'phpreport');
+define('DB_HOST', 'localhost');
+define('DB_PORT', 5432);
+
+
+define('PHPREPORT_ROOT', __DIR__ . '/../../');
+include_once(PHPREPORT_ROOT . '/util/DBPostgres.php');
+
+$link = pg_connect("host=".DB_HOST.
+	" port=".DB_PORT.
+	" user=".DB_USER.
+	" dbname=".DB_NAME.
+	" password=".DB_PASSWORD);
+if (!$link) error_log("ERROR: Could not connect to DB");
+
+$result = pg_query($link, $query = "SELECT customerid, projectid FROM requests");
+if ($result == NULL) error_log("ERROR: Could not run query: $query");
+$frequency = array();
+
+while ($row=pg_fetch_array($result, NULL, PGSQL_ASSOC)) {
+	$frequency[$row['projectid']][$row['customerid']] = true;
```

Comment:
`$frequency` is not a good name for this variable, it has no relation with its contents. Also, you are using the keys to contain the customer ids, and discarding the content of the variable, but I think it's cleaner to use arrays like this:
```
$frequency[$row['projectid']][] = $row['customerid'];
```
It will make your array look like:
```
  [256]=>
  array(1) {
    [0]=>
    string(2) "33"
  }
  [257]=>
  array(17) {
    [0]=>
    string(2) "33"
    [1]=>
    string(2) "39"
    [2]=>
    string(2) "34"
    [3]=>
    string(2) "41"
    [4]=>
    string(1) "4"
    [5]=>
    string(2) "16"
    [6]=>
    string(2) "19"
    [7]=>
    string(2) "35"
    [8]=>
    string(2) "31"
    [9]=>
    string(2) "11"
    [10]=>
    string(2) "38"
    [11]=>
    string(1) "5"
    [12]=>
    string(2) "42"
    [13]=>
    string(2) "10"
    [14]=>
    string(1) "7"
    [15]=>
    string(1) "9"
    [16]=>
    string(2) "14"
  }